config.yaml

```diff
@@ -3,45 +3,93 @@ options:
     default: True
     type: boolean
     description: |
-      Limit requested identifiers (hostnames, IPs and OIDs) to first requester
+      This filter is disabled by default, and can be enabled by setting the following config:
```

Comment:
No need to tell users how they can enable configs, this is applicable to every config values in the juju ecosystem. This applies at a couple of places in this PR.